hack/build-images.sh

```diff
@@ -56,8 +62,9 @@ function image {
   docker tag -f $1:latest $1:${OS_RELEASE_COMMIT}
 }
 
-# images that depend on scratch
+# images that depend on scratch / centos
 image openshift/origin-pod                   images/pod
+image openshift/openvswitch                  images/openvswitch
```

Comment:
Does the openvswitch binary image depend on anything outside of its directory?  If not, in a follow up let's remove it from here and put it in the "base-images" flow.